Library/Homebrew/test.rb

```diff
@@ -17,6 +17,15 @@
   ENV.extend(Stdenv)
   ENV.setup_build_environment
 
+  # This ensures all executables in test do are called with a full path, i.e.
+  # bin/"wget", or Formula["openssl"].opt_bin/"openssl" rather than just "wget".
+  # It's very likely to break tests, so phase it in via CI rather than user
```

Comment:
Or we could stop chasing people around about this altogether and prepend the bin paths for keg only formulae and the sbin paths for all formula.